src/data/game.json

```diff
@@ -22,5 +23,6 @@
         {
             "random": 3
         }
-    ]
-}
\ No newline at end of file
+    ],
+    "death_link": true

```

Comment:
lets maybe not have it enabled by default? I feel like it should be something the creator has to choose to turn on